---- Minecraft Crash Report ----
// Daisy, daisy...

Time: 09.04.25 19:40
Description: Exception generating new chunk

java.lang.ArrayIndexOutOfBoundsException: 7
	at com.github.krz19.WorldGen.BuildingGen.generateFeature(BuildingGen.java:76)
	at com.github.krz19.WorldGen.CitiesGenChunkProvider.fillPlatform(CitiesGenChunkProvider.java:115)
	at com.github.krz19.WorldGen.CitiesGenChunkProvider.provideChunk(CitiesGenChunkProvider.java:43)
	at net.minecraft.world.gen.ChunkProviderServer.originalLoadChunk(ChunkProviderServer.java:190)
	at net.minecraft.world.gen.ChunkProviderServer.loadChunk(ChunkProviderServer.java:149)
	at net.minecraft.world.gen.ChunkProviderServer.loadChunk(ChunkProviderServer.java:119)
	at net.minecraft.server.MinecraftServer.initialWorldChunkLoad(MinecraftServer.java:305)
	at net.minecraft.server.integrated.IntegratedServer.loadAllWorlds(IntegratedServer.java:79)
	at net.minecraft.server.integrated.IntegratedServer.startServer(IntegratedServer.java:96)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:445)
	at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:752)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Stacktrace:
	at com.github.krz19.WorldGen.BuildingGen.generateFeature(BuildingGen.java:76)
	at com.github.krz19.WorldGen.CitiesGenChunkProvider.fillPlatform(CitiesGenChunkProvider.java:115)
	at com.github.krz19.WorldGen.CitiesGenChunkProvider.provideChunk(CitiesGenChunkProvider.java:43)

-- Chunk to be generated --
Details:
	Location: -1,23
	Position hash: 103079215103
	Generator: RandomLevelSource
Stacktrace:
	at net.minecraft.world.gen.ChunkProviderServer.originalLoadChunk(ChunkProviderServer.java:190)
	at net.minecraft.world.gen.ChunkProviderServer.loadChunk(ChunkProviderServer.java:149)
	at net.minecraft.world.gen.ChunkProviderServer.loadChunk(ChunkProviderServer.java:119)
	at net.minecraft.server.MinecraftServer.initialWorldChunkLoad(MinecraftServer.java:305)
	at net.minecraft.server.integrated.IntegratedServer.loadAllWorlds(IntegratedServer.java:79)
	at net.minecraft.server.integrated.IntegratedServer.startServer(IntegratedServer.java:96)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:445)
	at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:752)

-- System Details --
Details:
	Minecraft Version: 1.7.10
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_422, BellSoft
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), BellSoft
	Memory: 516395360 bytes (492 MB) / 762314752 bytes (727 MB) up to 2837446656 bytes (2706 MB)
	JVM Flags: 0 total; 
	AABB Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	IntCache: cache: 0, tcache: 0, allocated: 13, tallocated: 90
	FML: MCP v9.05 FML v7.10.99.99 Minecraft Forge 10.13.4.1614 4 mods loaded, 4 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
	UCHIJAA	mcp{9.05} [Minecraft Coder Pack] (minecraft.jar) 
	UCHIJAA	FML{7.10.99.99} [Forge Mod Loader] (forgeSrc-1.7.10-10.13.4.1614-1.7.10.jar) 
	UCHIJAA	Forge{10.13.4.1614} [Minecraft Forge] (forgeSrc-1.7.10-10.13.4.1614-1.7.10.jar) 
	UCHIJAA	CityGenMod{ALPHA 0.0.001} [City Generation Mod] (roadmod-1.0.jar) 
	GL info: ~~ERROR~~ RuntimeException: No OpenGL context found in the current thread.
	Profiler Position: N/A (disabled)
	Vec3 Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	Player Count: 0 / 8; []
	Type: Integrated Server (map_client.txt)
	Is Modded: Definitely; Client brand changed to 'fml,forge'